src/Entity/Comment.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace App\Entity;
+
+use DateTime;
+use DateTimeInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\OneToMany;
+use JMS\Serializer\Annotation\Groups;
+use JMS\Serializer\Annotation as Serializer;
+use Swagger\Annotations as SWG;
+
+/**
+ * @ORM\HasLifecycleCallbacks
+ * @ORM\Entity(repositoryClass="App\Repository\CommentRepository")
+ * @Serializer\ExclusionPolicy("all")
+ */
+class Comment
+{
+    /**
+     * @ORM\Id()
+     * @OneToMany(targetEntity="Comment", mappedBy="reply")
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"Comment"})
+     * @SWG\Property()
+     */
+    private $id;
+
+    /**
+     * Activity id.
+     * @ORM\ManyToOne(targetEntity="Activity")
+     * @ORM\JoinColumn(onDelete="CASCADE")
+     * @Serializer\Expose()
+     * @SWG\Property()
+     */
+    private $activity;
+
+    /**
+     * User id.
+     * @ORM\ManyToOne(targetEntity="User")
+     * @ORM\JoinColumn(onDelete="CASCADE")
```

Comment:
This will create problems. Test case to try:
1. Create a comment with user1
2. Reply to that comment with user2
3. Remove user1